src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+:root {
+  box-sizing: border-box;
+}
+
+.search-form {
+  display: block;
+  position: relative;
+  margin-top: 20px;
+}
+
+.search-form__input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+
+  background-color: #fff;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+
+}
+
+.search-form__input::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+  text-shadow: none;
+}
+
+.search-form__input:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+  outline: none;
+  border: 1px solid #e1e7ed;
+}
+
+.search-form__input:focus {
+  outline: none;
+  font-weight: 800;
+  text-shadow: 1px 4px 4px rgba(0, 0, 0, 0.25);
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-form__input--small {
+  height: 42px;
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.search-form__image {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+
+  height: 19px;
+  width: 19px;
+}
+
+.search-form__image--small {
+  top: 15px;
```

Comment:
Use the reusable way to center the image vertically https://www.w3schools.com/howto/howto_css_center-vertical.asp